.github/workflows/release.yaml

```diff
@@ -5,7 +5,13 @@ on:
       - "release"
       - "next"
 jobs:
+  test:
+    uses: ./.github/workflows/go-test.yaml
```

Comment:
You'd also need a 'workflow_call:' in go-test.yml. Check python ci.yml for reference.

Reply:
ah ,right.

Reply:
updated.